mkosi/backend.py

```diff
@@ -210,6 +210,7 @@ class ManifestFormat(Parseable, enum.Enum):
 class Partition:
     size: int
     number: int
+    name: str
```

Comment:
this is a descriptive text we only use for log messages, no? should be called `description` or `comment` or so maybe?